src/spark/gedi_download_pipeline.py

```diff
@@ -348,7 +298,23 @@ def exec_spark(
         print("Unable to locate file {}".format(shapefile))
         exit(1)
     shp = gpd.read_file(shapefile)
-    shp = _check_and_format_shape(shp)
+    try:
+        try:
+            shp = check_and_format_shape(shp)
+        except DetailError:
+            input(
+                (
+                    "The NASA API can only accept up to 5000 vertices in a single shape,\n"
+                    "but the shape you supplied has {} vertices.\n"
+                    "If you would like to automatically simplify this shape to its\n"
+                    "bounding box, press ENTER, otherwise Ctrl-C to quit."
+                ).format(n_coords)
```

Comment:
I think this variable is no longer in scope?

Reply:
Good catch! Made it a parameter on the exception so we can still display list.